restdocs-api-spec-openapi3-generator/src/test/kotlin/com/epages/restdocs/apispec/openapi3/OpenApi3GeneratorTest.kt

```diff
@@ -730,6 +778,30 @@ class OpenApi3GeneratorTest {
         )
     }
 
+    private fun getProductRequestAsFormData(method: HTTPMethod, getSecurityRequirement: () -> SecurityRequirements = ::getOAuth2SecurityRequirement): RequestModel {
```

Comment:
The `get` in `getProductRequestAsFormData` comes from the HTTP method GET. Thus I'd just name it
`productRequestAsFormData` (the method is defined by the first parameter)